quickwit-cli/Cargo.toml

```diff
@@ -46,7 +46,7 @@ tempfile = "3"
 chrono = "0.4"
 humansize = "1.1.1"
 openssl-probe = { version = "0.1.4", optional = true }
-itertools = "0.10.1"
+itertools = "0.10"
```

Comment:
Contrary to what i might have said before.
I recommend leaving it at 0.10.1. 

First
`0.10.1` actually stands for `^0.10.1`

So `0.10.2` is actually matching the requirement.
Second, there might be an actual good reason to put 0.10.1 and not 0.10.0.

For instance, 0.10.1 may have added extra functionality compared to 0.10.
So it is fairly possible that quickwit does not compile with 0.10.0.

A contributor actually spotted a couple of such example and convinced me to change our dependancy version for that reason. (I don't know if itertools was one of them or not).

Reply:
Sounds good to me. Will revert.